src/profiler.cpp

```diff
@@ -1556,9 +1568,14 @@ void Profiler::dumpText(Writer& out, Arguments& args) {
             if (trace == NULL || counter == 0) continue;
 
             total_counter += counter;
-            if (trace->num_frames == 0 || excludeTrace(&fn, trace)) continue;
+            if (trace->num_frames == 0) continue;
+            if (excludeTrace(&fn, trace)) {
+                filtered_sample_count++;
+                continue;
+            }
             samples.push_back(it->second);
         }
+        logIfNoSamples(args, filtered_sample_count);
```

Comment:
Text output already includes the number of collected samples in the summary. I'm not sure if it's worth logging in addition to that. I'd say log matters mostly for `collapsed` and `flamegraph` output. Let me know if you think otherwise.

Reply:
That's correct. But this information allows user to identify the cases mentioned above (no collected samples vs. no printed samples).